src/main/java/org/mule/extension/http/internal/request/HttpRequestFactory.java

```diff
@@ -143,10 +143,10 @@ public HttpRequest create(HttpRequesterRequestBuilder requestBuilder, HttpAuthen
     return builder.build();
   }
 
-  private ParameterMap toParameterMap(Map<String, String> map) {
-    ParameterMap parameterMap = new ParameterMap();
-    map.forEach(parameterMap::put);
-    return parameterMap;
+  private MultiMap<String, String> toMultiMap(Map<String, String> map) {
```

Comment:
Seems like it, though it also checks for collection values. In any case, this will be removed in another upcoming task which will turn the HTTP parameters into MultiMaps themselves.